tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,3 @@
+export const trimWhiteSpaces = (string) => {
+  return string.replace(/\s/g, '');
```

Comment:
There's a lot of ways to do such a thing. I wanted to practice with RegEx because I used it earlier. Also, I found this website really helpful - https://regex101.com.